### PR TITLE
Updated documentation links and other minor fixes

### PR DESCRIPTION
**Pull request by ahcorde**
Updated documentation links and other minor fixes
Signed-off-by: Alejandro Hernández <ahcorde@gmail.com>

### CODE REVIEW

File: tutorials/02_configuration.md [hunk 1]
```diff
@@ -62,13 +62,13 @@ mkdir /tmp/conf_tutorial && cd /tmp/conf_tutorial
 Download the file `download.cc` and save it under `/tmp/conf_tutorial`:
 
 ```
-wget https://github.com/ignitionrobotics/ign-fuel-tools/raw/ign-fuel-tools4/example/download.cc
+wget https://github.com/ignitionrobotics/ign-fuel-tools/raw/master/example/download.cc
```
Comment (mjcarroll) [inline tutorials/02_configuration.md:65]: This should probably be `ign-fuel-tools5` for consistency, right?

**Review by mjcarroll — COMMENTED** [review]
One question on branch names.  I think that codecov can be safely ignored.

File: tutorials/02_configuration.md (hunk 1 above)
Comment (ahcorde) [reply tutorials/02_configuration.md:65]: There is no branch for `ign-fuel-tools5`.

**Review by nkoenig — APPROVED** [review]

File: tutorials/02_configuration.md (hunk 1 above)
Comment (nkoenig) [reply tutorials/02_configuration.md]: Done in cee8927941175ab920dae47b5f3758eba2d1d911

**Review by chapulina — APPROVED** [review]

**Pull request merged**